### PR TITLE
Bogdan

### PR DESCRIPTION
**Pull request by bogdanishere**
Changes according to the specifications.

### CODE REVIEW

File: server/package.json [hunk 1]
```diff
@@ -21,6 +21,7 @@
     "pg": "^8.12.0",
     "pg-hstore": "^2.3.4",
     "sequelize": "^6.37.3",
+    "server": "file:",
```
Comment (cristim67) [inline server/package.json:24]: ???

File: client/tailwind.config.js [hunk 2]
```diff
@@ -1,6 +1,7 @@
 /** @type {import('tailwindcss').Config} */
 export default {
   content: ["./index.html", "./src/**/*.{js,ts,jsx,tsx}"],
+
```
Comment (cristim67) [inline client/tailwind.config.js:4]: ???

File: client/package.json [hunk 3]
```diff
@@ -10,9 +10,13 @@
     "preview": "vite preview"
   },
   "dependencies": {
+    "@tanstack/react-query": "^5.54.1",
+    "client": "file:",
```
Comment (cristim67) [inline client/package.json:14]: ???

File: server/package.json (hunk 1 above)
Comment (bogdanishere) [reply server/package.json:24]: This dependency (server: "file") was automatically added by my editor. I didn’t manually make any changes, but it seems to be due to some settings or auto-completion in my IDE. If we don’t need it, we can safely remove it.

File: client/package.json (hunk 3 above)
Comment (bogdanishere) [reply client/package.json:14]: This dependency (client: "file") was automatically added by my editor. I didn’t manually make any changes, but it seems to be due to some settings or auto-completion in my IDE. If we don’t need it, we can safely remove it.

File: server/package.json (hunk 1 above)
Comment (cristim67) [reply server/package.json:24]: if i take this branch and run `npm install`, that command fails.

File: client/tailwind.config.js (hunk 2 above)
Comment (bogdanishere) [reply client/tailwind.config.js:4]: I didn't make any changes here compared to the main branch.

**Pull request closed without merging**